core/src/main/java/com/rim/IceField/PlayerBaseTest.java

```diff
@@ -216,5 +217,105 @@ void eatFood() {
         assertEquals(5,p1.heatLevel);
      }
 
+     @Test
+     void polarSkill(){
+        //Test-case 15: Polar explorer skill
+         PlayerBase  p1 = new PolarExplorer();
```

Comment:
Do you have the same setup in all of the tests? maybe you could use something like before each or before all, to create one setup.

Reply:
It really depends on the test case: number of chatacters, type of chaarcters, if we use game as an instances, but there are some common things of course, so before is a good suggestion. Will add that.